plugins/cilium-docker/driver/driver.go

```diff
@@ -77,19 +82,28 @@ func newLibnetworkRoute(route route.Route) api.StaticRoute {
 // NewDriver creates and returns a new Driver for the given API URL.
 // If url is nil then use SockPath provided by CILIUM_SOCK
 // or the cilium default SockPath
-func NewDriver(url string) (Driver, error) {
+func NewDriver(ciliumSockPath, dockerHostPath string) (Driver, error) {
 
-	if url == "" {
-		url = client.DefaultSockPath()
+	if ciliumSockPath == "" {
+		ciliumSockPath = client.DefaultSockPath()
 	}
 
-	scopedLog := log.WithField("url", url)
-	c, err := client.NewClient(url)
+	scopedLog := log.WithField("ciliumSockPath", ciliumSockPath)
+	c, err := client.NewClient(ciliumSockPath)
 	if err != nil {
 		scopedLog.WithError(err).Fatal("Error while starting cilium-client")
 	}
 
-	d := &driver{client: c}
+	scopedLog = scopedLog.WithField("dockerHostPath", dockerHostPath)
+	dockerCli, err := dockerCliAPI.NewClientWithOpts(
+		dockerCliAPI.WithVersion("v1.21"),
```

Comment:
Any specific need for this version? If so, hardcode it into a constant and state why it is used for self-documenting code.

Reply:
It's the same being used today. https://github.com/cilium/cilium/blob/cf7e154e73f4ca76faa28357753b7af6dfb282f8/pkg/workloads/docker.go#L103